test/sdp.jl

```diff
@@ -2,6 +2,8 @@
 # Test SDDP functions
 ################################################################################
 using FactCheck, StochDynamicProgramming
+include("../src/SDPutils.jl")
```

Comment:
instead of including the file here, should be able to do `using StochDynamicProgramming.SDPutils`